functions/compute_fieldmap.py

```diff
@@ -119,9 +119,16 @@ def compute_fieldmap(input_file, output_file):
         None
     """
     if is_nifti(input_file):
+        tic = time()
```

Comment:
look into time.perf_counter(), might be more appropriate. Also maybe rename to start_time and end_time

Reply:
Thx for the feedback!